libs/app-shell/src/lib/api/extensions/lookup/plugin-descriptor.model.ts

```diff
@@ -1,31 +1,50 @@
-export interface PluginDescriptor {
+export interface AngularIvyComponentDescriptor {
+    /** Uniq name of module in plugin */
     name: string;
-    provider: string
-    version: string;
-    sinceVersion: string;
-    untilVersion: string;
-    remoteEntry: string;
-    remoteName: string;
-    remoteRoute: string;
+    /** Type of a plugin module */
+    type: 'angular-ivy-component';
+    /** Exposed Angular Module */
     exposedModule: string;
-    displayName: string;
+    /** Component name we are planning to inject */
     componentName: string;
-
-    type: Scope;
-    category: string;
-    hasRoutes: boolean;
-    changeNotes: string;
+    /** most likely this property will be removed, but as for this this is a way to register route for a component*/
+    route?: string;
 }
 
+export interface IframePageDescriptor {
+    /** Uniq name of module in plugin */
+    name: string;
+    /** Type of a plugin module */
+    type: 'iframe';
+    /** link to html relative to URI, where uri+html should give a valid URL */
+    html: string
+}
 
-/**
- * The idea behind the scope is that there are components that are instantiated at the time we use Page launcher.
- * But there could be a situation, where we need to register a PLUGIN at the bootstrap at the lauch time, there we
- * have scope Application.
- * Not implemented yet default value should be as Page
- */
-export enum Scope {
-    Application = 1,
-    Page
+export interface AngularElementComponentDescriptor {
```

Comment:
This should not be an Angular Element but more CustomElementComponentDesriptor, this is our current demo, what I had. we dont have demo for AngularElements

Reply:
It's not final, I will check what we actually have in bundles